modules/gapi/include/opencv2/gapi/stereo.hpp

```diff
@@ -32,11 +45,16 @@ G_TYPED_KERNEL(GStereo, <GMat(GMat, GMat, const StereoOutputFormat)>, "org.openc
         GAPI_Assert(right.depth == CV_8U);
 
         switch(of) {
+            case StereoOutputFormat::DEPTH_F16:
             case StereoOutputFormat::DEPTH_FLOAT16:
                 return left.withDepth(CV_16FC1);
+            case StereoOutputFormat::DEPTH_F32:
             case StereoOutputFormat::DEPTH_FLOAT32:
                 return left.withDepth(CV_32FC1);
+            case StereoOutputFormat::DISPARITY_Q16_10_5:
             case StereoOutputFormat::DISPARITY_FIXED16_11_5:
+
+            case StereoOutputFormat::DISPARITY_Q16_11_4:
```

Comment:
I hope this changes are consistent (Part2). And for bottom changes.

Reply:
Its more relevant `StereoOutputFormat` state names